weed/filer/mysql_store/filer_mapping.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS `filer_mapping` (
```

Comment:
This file is not needed since you are auto creating the table.
You can add a README.txt file and add the SQL there.

Also, add example json config files in the README.